content/en/docs/architecture/release-gating.md

```diff
@@ -4,60 +4,137 @@ description: An overview of how CI jobs are configured to gate OpenShift release
-#### Show the Job is Stable
-After 20 jobs have run and the success rate is 100%, the job will be considered stable enough to be considered for inclusion to the nightly release blocking suite. Any release blocking test suites must have an extremely low flake rate. As noted in our [Build Cop](https://docs.google.com/document/d/117_0UE5jJI_MyI5ugy1psn0Ls6fWCu-Y9jiZZPM4qzw/edit?ts=5c7d4ca0#heading=h.7i59o09vqwdg) documentation, our targets for all release jobs are 100% success.
-#### Finalize the Job Name
-In order to make sure the job is reporting on the build cop dashboards so build cops are alerted if it starts failing, rename the job to begin with release- and not canary-, choosing the final name of the suite. This name change will include your job in the release job metrics and dashboards.
-#### Add the Job to the Release Gating Suite as Optional
-Edit the release controller’s configuration files to add your suite by name, setting optional: true. For example, the 4.2 release gating suites are defined [here](https://github.com/openshift/release/blob/master/core-services/release-controller/_releases/release-ocp-4.2.json).
-#### Mark new Job as Required in the Release Gating Suite
-Once the optional job is running successfully on OCP nightlies for a week, reconfigure your job to be required in the release and assign a release lead to approve the pull request to openshift/release.
-#### Sign Up for Build Cop Rotations
-Congratulations! Your job is blocking OpenShift nightly releases. The members of your team are now required to sign up to be part of the [Build Cop rotation](https://docs.google.com/document/d/117_0UE5jJI_MyI5ugy1psn0Ls6fWCu-Y9jiZZPM4qzw/edit?ts=5c7d4ca0#). Contact Eric Paris to be included in the schedule. You will have an OpenShift group lead assigned to work with you during your team’s first rotation.
+
+Our goal of weekly z-stream releases necessitates that release-gating tests provide stable and unambiguous signal about
+regressions in the product or the projects and projects on top. To provide this level of stability and observability,
+any new release-blocking tests must follow the below steps:
+
+#### Add A Periodic Job
+
+Create [a new multi-stage job](/docs/architecture/step-registry/) on top of latest OCP that runs as a periodic job. The
+job should reuse as much of the existing step registry as possible, and must include either the standard gather and
+gather-extra steps from the step registry, or custom steps (in rare cases) that gather as much of the same data as possible.
+
+This does not block the release but you can run it often enough to get a decent signal. At least once a day, but every 6
+or 12 hours is better.
+
+The job name should follow the [naming guidelines](/docs/how-tos/naming-your-ci-jobs/).
```

Comment:
`naming-your-ci-job` does not exist

Reply:
it's there on master branch

Reply:
It'll work when it's live: https://docs.ci.openshift.org/docs/how-tos/naming-your-ci-jobs/ exists